src/Extensions/TimedScopes/Internal/TimedScopeEventSource.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics.Tracing;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions;
+using Microsoft.Omex.Extensions.Logging.TimedScopes;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	[EventSource(Name = "Microsoft-OMEX-TimedScopes")]
+	internal sealed class TimedScopeEventSource : EventSource
+	{
+		public TimedScopeEventSource(ILogger<TimedScopeEventSource> logger)
+		{
+			m_logger = logger;
+			m_logCategory = typeof(TimedScopeEventSource).FullName ?? nameof(TimedScopeEventSource);
+		}
+
+
+		[NonEvent]
+		public void LogEvent(
+			string name,
+			string subtype,
+			string metadata,
+			string userHash,
+			string serviceName,
+			TimedScopeResult result,
+			string correlationId,
+			double durationMs,
+			bool isTransaction)
+		{
+			if (!IsEnabled())
+			{
+				return;
+			}
+
+			string nameAsString = SanitizeString(name, nameof(name), name);
+			string subTypeAsString = SanitizeString(subtype, nameof(subtype), name);
+			string metaDataAsString = SanitizeString(metadata, nameof(metadata), name);
+			string userHashAsString = SanitizeString(userHash, nameof(userHash), name);
+			string serviceNameAsString = SanitizeString(serviceName, nameof(serviceName), name);
+			string correlationIdAsString = SanitizeString(correlationId, nameof(correlationId), name);
+			string resultAsString = result.ToString();
+			long durationMsAsLong = Convert.ToInt64(durationMs);
```

Comment:
>ToInt64 [](start = 35, length = 7)

Add `CultureInfo.InvariantCulture` as a parameter to this.

Reply:
Fixed, thanks